fastapi_security/cli.py

```diff
@@ -0,0 +1,93 @@
+"""fastapi_security command-line interface"""
+
+import argparse
+import sys
+import textwrap
+from getpass import getpass
+from typing import Optional, Sequence, Text
+
+from fastapi_security.basic import generate_digest
+
+
+def _wrap_paragraphs(s):
+    paragraphs = s.strip().split("\n\n")
+    wrapped_paragraphs = [
+        "\n".join(textwrap.wrap(paragraph)) for paragraph in paragraphs
+    ]
+    return "\n\n".join(wrapped_paragraphs)
+
+
+main_parser = argparse.ArgumentParser(
+    description=_wrap_paragraphs(__doc__),
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+)
```

Comment:
`prog` needs to be added for unit test change suggestion I'm making below to work.

```suggestion
main_parser = argparse.ArgumentParser(
    prog="fastapi-security",
    description=_wrap_paragraphs(__doc__),
    formatter_class=argparse.RawDescriptionHelpFormatter,
)
```